.snyk

```diff
@@ -58,7 +58,14 @@ ignore:
           Version 3.2 of this package contains an incomplete fix,
           which might help reduce the chances of this vulnerability 
           being exploited.
-        expires: 2021-05-05T06:00:00.000Z
+        expires: 2021-06-01T06:00:00.000Z
+
+  SNYK-PYTHON-BABEL-1278589:
+    - '*':
+        reason: >-
+          Wait for upstream CKAN to update Babel requirement to 2.9.1 or higher.
```

Comment:
You can add that CloudFront+WAF mitigates this issue. I tried locally and was unable to get the path traversal to even hit CKAN.